cmd/hypper/pull_test.go

```diff
@@ -0,0 +1,199 @@
+// +build !windows
```

Comment:
```suggestion
```

You are gonna hate me.. :sweat_smile:

Reply:
It would be a plus to have this being run on windows as well :)

Reply:
Im trying to remember why I set this....

Reply:
yeah this requires changes on the repotest package of helm, which goes out of scope. I cannot support windows unless there is changes upstream to make it sure it works under windows as well :)